langserver/completion.go

```diff
@@ -102,9 +103,33 @@ func (s *server) completeMetricName(ctx context.Context, completions *[]protocol
 	var allNames model.LabelValues
 
 	if api != nil {
-		var err error
+		// isCompatible checks if language server has access to new metadata APIs.
+		var isCompatible = false // nolint:ineffassign
```

Comment:
Since almost exactly the same code is also present in `hover.go` you could unify it into one function.